packages/server/lib/server-base.ts

```diff
@@ -320,7 +326,7 @@ export abstract class ServerBase<TSocket extends SocketE2E | SocketCt> {
 
   createNetworkProxy ({ config, remoteStates, requestedWithAndCredentialManager, shouldCorrelatePreRequests }) {
     const getFileServerToken = () => {
-      return this._fileServer.token
+      return this._fileServer?.token
```

Comment:
Right, so this was doing the classic "could not get `token` of `undefined` error, but manifesting in some other bizarre way (perhaps the error middleware was called after the request already finished?)

Reply:
Correct, the "token" error occurred and we called our `onError` handler which then tried to queue the request up to be retried which called `this.next` a second time. This second invocation threw a separate error which squashed the original